src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventAttachments/groupEventAttachmentsFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import omit from 'lodash/omit';
+import xor from 'lodash/xor';
+import {Link, withRouter} from 'react-router';
+import {WithRouterProps} from 'react-router/lib/withRouter';
+
+import {t} from 'app/locale';
+
+const GroupEventAttachmentsFilter = (props: WithRouterProps) => {
+  const {query, pathname} = props.location;
+  const {types} = query;
+  const onlyCrashReportTypes = ['event.minidump', 'event.applecrashreport'];
+  const allAttachmentsQuery = omit(query, 'types');
+  const onlyCrashReportsQuery = {
+    ...query,
+    types: onlyCrashReportTypes,
+  };
+
+  return (
+    <div className="text-right">
```

Comment:
I'm assuming this floats right, instead, let's instead add a local styled component using `emotion` and make it a flexbox that justifies to `flex-end`.

Reply:
awesome, thanks